common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -356,6 +356,10 @@ def get_data_for_vertical(self, vertical):
                     return (group.name, group.id)
         return (None, None)
 
+    @property
+    def tooltip_content(self):
+        return getattr(self.child, 'tooltip_content', '') if self.child else ''
```

Comment:
If we're trying to return `''` when `self` doesn't have a `child` attribute, you'll need something like:

    `getattr(getattr(self, 'child', ''), 'tooltip_content', '')`.  

Or:

    try:
        return self.child.tooltip_content
    except AttributeError:
        return ''